src/main/java/com/rls/lms/controllers/UserController.java

```diff
@@ -0,0 +1,34 @@
+package com.rls.lms.controllers;
+
+import com.rls.lms.models.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import com.rls.lms.repositories.UserRepository;
+
+@RestController    // This means that this class is a Controller
+@RequestMapping(path="/api") // This means URL's start with /demo (after Application path)
+public class UserController {
+    @Autowired // This means to get the bean called userRepository
+    // Which is auto-generated by Spring, we will use it to handle the data
+    private UserRepository userRepository;
+
+    @GetMapping(path="/add") // Map ONLY GET Requests
```

Comment:
it would be better to use **/v1/users** as path and method as **POST** to create new user.

Reply:
This was in a previous commit. So we can omit this one.